modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache

```diff
@@ -148,8 +148,23 @@ public class ApiClient {
    * Ctor.
    */
   public ApiClient() {
-    builder = HttpClient.newBuilder();
-    mapper = new ObjectMapper();
+    this(createDefaultHttpClientBuilder(), createDefaultObjectMapper(), getDefaultBaseUri());
+  }
+
+  /**
+   * Ctor.
+   */
+  public ApiClient(HttpClient.Builder builder, ObjectMapper mapper, String basePath) {
```

Comment:
I suggest renaming the `basePath` parameter to `baseUri` so it won't be confused with `this.basePath` field which only stores the "RawPath".

Reply:
fixed